### PR TITLE
Missing topics 2.3

### PR DESCRIPTION
**Pull request by pinkeshmars**
### Description
1. YouTubePlayer
2. VideoPlayer
3. Blur
4. Braintree
5. Firebase Delete User
6. Firebase Update Email
7. Colors
8. Typography
9. Theme widgets
10. MediaDisplay widget

Linear ticket and [magic word](https://linear.app/docs/github#link-prs) Fixes DEVR-573 

## Type of change
- [ ] Typo fix 
- [x] New feature 
- [ ] Enhancement to current docs
- [ ] Removed outdated references
- [ ] Update assets

### CODE REVIEW

File: docs/ff-concepts/design-system/design-system.md [hunk 1]
```diff
@@ -333,9 +335,700 @@ You can modify its color, background color, and stroke width. In the 'Preview' s
...
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+### Video guide
+
+If you prefer watching a video tutorial, here's the one for you:
```
Comment (PoojaB26) [inline docs/ff-concepts/design-system/design-system.md:832]: This video should be after the Custom Fonts section.

File: docs/ff-concepts/design-system/design-system.md [hunk 2]
```diff
@@ -333,9 +335,700 @@ You can modify its color, background color, and stroke width. In the 'Preview' s
...
+</div>
+<p></p>
+
+
+
 ## FAQs
 
 <details>
 <summary>How do I create a new design system if I don't see a button for it?</summary>
 <p>Currently, creating a design system requires you to have a team set up, even if you're working solo. You can quickly create a team by following the instructions outlined [here](../../accounts-billing/subscriptions/flutterflow-for-teams.md). We are working on a solution that will allow for the creation of design systems without needing a team setup. Stay tuned for updates!</p>
```
Comment (PoojaB26) [inline docs/ff-concepts/design-system/design-system.md:1027]: I dont think this is true today. I just tried on a Free plan account and I do have a Design System tab. Can you please recheck this and clarify?

File: docs/ff-integrations/payments/braintree.md [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+Braintree payments work on real Android devices or in emulators, and App Store purchases only work on real iOS devices. [This document](../../testing-deployment-publishing/running-your-app/local-run.md) has instructions on how to run your app on an Android or iOS device.
+
+:::info
+The Braintree Payments cannot be tested in Preview Mode, Test Mode, or Run Mode.
+:::
+
+To test your app before deployment:
+
+1. Download and run your project as described [here](../../testing-deployment-publishing/running-your-app/local-run.md).
+2. To test the purchase, you can use any of these [basic test card numbers](https://stripe.com/docs/testing#cards).
```
Comment (PoojaB26) [inline docs/ff-integrations/payments/braintree.md:228]: This links to stripe doc. Is that intentional?

**Review by PoojaB26 — CHANGES_REQUESTED** [review]
Few changes requested. I'm unable to see the Arcades, Idk if this is a network issue. Will check again after you've fixed these issues.

File: docs/ff-concepts/design-system/design-system.md (hunk 1 above)
Comment (pinkeshmars) [reply docs/ff-concepts/design-system/design-system.md]: Done!

File: docs/ff-concepts/design-system/design-system.md (hunk 2 above)
Comment (pinkeshmars) [reply docs/ff-concepts/design-system/design-system.md]: ok, I checked and it's available now. removing it.

File: docs/ff-integrations/authentication/firebase-auth/auth-actions.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
+---
+slug: /integrations/authentication/firebase/auth-actions
+title: Auth Actions
+description: Learn how to add Firebase Authentication actions in your FlutterFlow app.
+tags: [Auth Actions, Authentication, Firebase]
+sidebar_position: 1
+keywords: [FlutterFlow, Auth Actions, Authentication, Firebase]
+---
+# Authentication Actions
```
[the remark this answers is not in the document]
Comment (pinkeshmars) [reply docs/ff-integrations/authentication/firebase-auth/auth-actions.md:9]: Moved!

File: docs/ff-integrations/payments/braintree.md (hunk 3 above)
Comment (pinkeshmars) [reply docs/ff-integrations/payments/braintree.md:228]: Yes, they work for any payments provider.

File: docs/ff-integrations/payments/braintree.md [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+slug: /integrations/payments/braintree
+title: Braintree
+description: Learn how to integrate Braintree payments in your FlutterFlow app.
+tags: [Payments, Integration]
+keywords: [FlutterFlow, Braintree, Payments, Integration]
+---
+
+# Braintree
+
+You can accept payments in your app using Braintree (a service provided by PayPal) integration. This will also allow your users to pay directly using a credit card or using a service like PayPal, Google Pay, or Apple Pay
```
[the remark this answers is not in the document]
Comment (pinkeshmars) [reply docs/ff-integrations/payments/braintree.md]: ok, added.

**Comment by pinkeshmars** [comment]
@PoojaB26 all review comments are addressed. For the Design System, GIFs are compressed and updated in the same link.

**Comment by PoojaB26** [comment]
Rest are fine, but I wonder if MediaDisplay should be in Components or in VideoPlayer (since its not just videos). Could be its own page and later we can add info about its generated code. But you can figure out where should this standalone widget be kept, file handling or components. In either way, we should link to Image & VideoPlayer pages as links for talking about the rest of properties

**Comment by pinkeshmars** [comment]
> Rest are fine, but I wonder if MediaDisplay should be in Components or in VideoPlayer (since its not just videos). Could be its own page and later we can add info about its generated code. But you can figure out where should this standalone widget be kept, file handling or components. In either way, we should link to Image & VideoPlayer pages as links for talking about the rest of properties

Agree! It's moved to components now!

**Review by PoojaB26 — APPROVED** [review]

**Pull request merged**